documentation/troubleshooting.md

```diff
@@ -40,6 +40,10 @@ You can resolve this by [configuring an alternate tmpdir](bolt_transports_refere
 transport you're using, or by talking to your administrator about updating permissions for the
 directory.
 
+## My `plan/task show` output is not correct
+
+Try running Bolt with `--clear-cache` to reset the plan/task cache.
+
```

Comment:
Would it be helpful to add more detail? I was going to say something explaining the specific situation mentioned in the ticket, but I thought that might be too specific. Also, is this location in the file fine?

Reply:
Hm, this does feel a little sparse...Maybe something like:
```
Bolt caches task and plan metadata when listing plans and tasks, and only updates the cache when modules in the `<PROJECT DIRECTORY>/modules/` directory are modified. Try rerunning the command with `--clear-cache` to refresh the cache and update task and plan metadata.

The task or plan may also be present at a higher precedence on the modulepath. To verify that the specific task or plan is being loaded from the place you expect, run `bolt task show [task name]` or `bolt plan show [plan name]` to see the path the task or plan is loaded from.
```

Reply:
I like that wording a lot better :) Thank you!